http/handler_store.go

```diff
@@ -590,7 +590,7 @@ func (h *storeHandler) bindRoutes(router *Router) {
 	router.AddRoute("/backup/export", http.MethodPost, backupExport, h.BasicExport)
 	router.AddRoute("/backup/import", http.MethodPost, backupImport, h.BasicImport)
 	router.AddRoute("/collections", http.MethodGet, collectionDescribe, h.GetCollection)
-	router.AddRoute("/view", http.MethodPost, views, h.AddView)
+	router.AddRoute("/view", http.MethodPost, views, h.AddView) //TODO-FIX duplicate route
```

Comment:
todo: Please just remove it instead of the comment. The next line is simply overriting it.

Reply:
- [ ] will do just wanted to do it as a separate commit as it's out of scope of this PR and when I found this i was in a messy state haha

Reply:
done in commit: `PR(-): Remove Duplicate AddView Post Route`